src/ethereum.ts

```diff
@@ -47,10 +57,12 @@ export class EthereumHelpers implements Helpers {
    * @param rpc optional rpc endpoint to create eth chain client
    * @returns chain id of the connected chain
    */
-  static async create(network: "mainnet" | "testnet", rpcUri?: string): Promise<EthereumHelpers> {
+  static async create(
+    network: "mainnet" | "testnet",
+    rpcUri?: string,
+  ): Promise<EthereumHelpers> {
     const networkSettings = config.rpc.ethereum[network];
-    if (rpcUri !== undefined) 
-      networkSettings.rpc = rpcUri;
+    if (rpcUri !== undefined) networkSettings.rpc = rpcUri;
```

Comment:
Could you pls disable automatic linter? Especially that rule allowing `if(cond) clause` in a single line

Reply:
I have disabled the single-line if/else rule